Examples/MAX32655/I2C_EEPROM/main.c

```diff
@@ -73,28 +76,23 @@ int main(void)
     printf("Press ENTER key to Continue\n\n");
     getchar();
 
-    err = MXC_I2C_Init(I2C_MASTER, 1, 0);
+    err = Eeprom_24LC256_Init(&eeprom1_req, I2C_MASTER, EEPROM_24LC256_I2C_SLAVE_ADDR0,
```

Comment:
Thanks @kenan-balci, this looks good.  I have one more design suggestion:

- Remove the `&eeprom1_req` as a required parameter for the functions.

You can move the `eeprom_24lc256_req_t` struct to be _internal_ to the drivers so that the user doesn't need to pass the pointer in every time.

Since the user will not be interacting with this struct directly there's no need to expose it _unless_ you want to preserve the ability to interface with multiple EEPROMs at the same time.  If that's the case then you can leave it.

Reply:
My purpose is to give the driver the ability to communicate with more than one EEPROM, as you said. Up to 8 EEPROMs with different slave addresses can be connected to the same I2C bus. I tested communication with two EEPROMs, and the driver worked properly.